internal/postgres/config.go

```diff
@@ -53,6 +53,9 @@ safelink() (
 	// ReplicationUser is the PostgreSQL role that will be created by Patroni
 	// for streaming replication and for `pg_rewind`.
 	ReplicationUser = "_crunchyrepl"
+
+	// configMountPath is where to mount additional config files
+	configMountPath = "/etc/postgres"
```

Comment:
Are you thinking that `etc/postgres` will be too easily confused with `etc/postgresql`, or are you thinking that it's safe since it seems unused by the images otherwise?

Reply:
I think we're good with `/etc/postgres`. I don't know of anything else that would use the `postgres` or `postgresql` name out there.

Staying away from the directories used by packagers is probably good.